iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/bucket/DocOrgRelationAffOrgJoinerTest.java

```diff
@@ -0,0 +1,107 @@
+package eu.dnetlib.iis.wf.affmatching.bucket;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+
+import java.util.List;
+
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableList;
+
+import eu.dnetlib.iis.wf.affmatching.bucket.projectorg.model.AffMatchDocumentOrganization;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchAffiliation;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchOrganization;
+import scala.Tuple2;
+
+/**
+ * @author madryk
+ */
+public class DocOrgRelationAffOrgJoinerTest {
+
+    private DocOrgRelationAffOrgJoiner docOrgRelationAffOrgJoiner = new DocOrgRelationAffOrgJoiner();
+    
+    private JavaSparkContext sparkContext;
+    
+    
+    @Before
+    public void setup() {
+        
+        SparkConf conf = new SparkConf();
+        conf.setMaster("local");
+        conf.setAppName(getClass().getName());
+        
+        conf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer");
+        conf.set("spark.kryo.registrator", "pl.edu.icm.sparkutils.avro.AvroCompatibleKryoRegistrator");
+        
+        sparkContext = new JavaSparkContext(conf);
+        
+    }
+    
+    @After
+    public void cleanup() {
+        
+        if (sparkContext != null) {
+            sparkContext.close();
+        }
+        
+    }
+    
+    
+    //------------------------ TESTS --------------------------
+    
+    @Test
+    public void join() {
+        
+        // given
+        
+        JavaRDD<AffMatchAffiliation> affiliations = sparkContext.parallelize(ImmutableList.of(
+                new AffMatchAffiliation("DOC1", 1),
+                new AffMatchAffiliation("DOC1", 2),
+                new AffMatchAffiliation("DOC2", 1), // document not present in documentOrganizations rdd
```

Comment:
It's nice you have commented it this way as it simplifies reading test significantly.